$ diff -u <(build/binutils/powerpc-eabi-objdump -d build/GEDE01/obj/game/game_fn_8012E568.o) <(build/binutils/powerpc-eabi-objdump -d build/GEDE01/src/game/game_fn_8012E568.o)
# exit 1 means instruction streams differ.

--- /dev/fd/63	2026-10-03 11:12:00.200013721 -0400
+++ /dev/fd/62	2026-10-03 11:12:00.201013722 -0400
@@ -1,5 +1,5 @@
 
-build/GEDE01/obj/game/game_fn_8012E568.o:     file format elf32-powerpc
+build/GEDE01/src/game/game_fn_8012E568.o:     file format elf32-powerpc
 
 
 Disassembly of section .text:
@@ -8,369 +8,367 @@
    0:	94 21 ff b0 	stwu    r1,-80(r1)
    4:	7c 08 02 a6 	mflr    r0
    8:	90 01 00 54 	stw     r0,84(r1)
-   c:	be e1 00 2c 	stmw    r23,44(r1)
-  10:	7c 7b 1b 78 	mr      r27,r3
-  14:	7c 9c 23 78 	mr      r28,r4
-  18:	7c bd 2b 78 	mr      r29,r5
-  1c:	80 63 00 3c 	lwz     r3,60(r3)
-  20:	c0 3b 00 00 	lfs     f1,0(r27)
-  24:	c0 5b 00 04 	lfs     f2,4(r27)
-  28:	c0 7b 00 08 	lfs     f3,8(r27)
-  2c:	48 00 00 01 	bl      2c <fn_8012E568+0x2c>
-  30:	80 9b 02 40 	lwz     r4,576(r27)
-  34:	57 80 10 3a 	slwi    r0,r28,2
-  38:	7c 7f 1b 78 	mr      r31,r3
-  3c:	7c 84 00 2e 	lwzx    r4,r4,r0
-  40:	83 c4 00 04 	lwz     r30,4(r4)
-  44:	48 00 00 01 	bl      44 <fn_8012E568+0x44>
-  48:	80 1b 02 94 	lwz     r0,660(r27)
-  4c:	7f 63 db 78 	mr      r3,r27
-  50:	38 80 00 00 	li      r4,0
-  54:	90 1f 02 94 	stw     r0,660(r31)
-  58:	48 00 00 01 	bl      58 <fn_8012E568+0x58>
-  5c:	80 7b 00 2c 	lwz     r3,44(r27)
-  60:	28 1d 00 00 	cmplwi  r29,0
-  64:	80 1b 00 30 	lwz     r0,48(r27)
-  68:	90 7f 00 2c 	stw     r3,44(r31)
-  6c:	90 1f 00 30 	stw     r0,48(r31)
-  70:	80 7b 00 34 	lwz     r3,52(r27)
-  74:	80 1b 00 38 	lwz     r0,56(r27)
-  78:	90 7f 00 34 	stw     r3,52(r31)
-  7c:	90 1f 00 38 	stw     r0,56(r31)
-  80:	80 1b 02 50 	lwz     r0,592(r27)
-  84:	90 1f 02 50 	stw     r0,592(r31)
-  88:	80 1b 02 44 	lwz     r0,580(r27)
-  8c:	90 1f 02 44 	stw     r0,580(r31)
-  90:	41 82 00 1c 	beq     ac <fn_8012E568+0xac>
-  94:	80 7d 00 00 	lwz     r3,0(r29)
-  98:	80 1d 00 04 	lwz     r0,4(r29)
-  9c:	90 7f 00 0c 	stw     r3,12(r31)
-  a0:	90 1f 00 10 	stw     r0,16(r31)
-  a4:	80 1d 00 08 	lwz     r0,8(r29)
-  a8:	90 1f 00 14 	stw     r0,20(r31)
-  ac:	38 60 00 00 	li      r3,0
-  b0:	38 00 00 11 	li      r0,17
-  b4:	7c 64 1b 78 	mr      r4,r3
-  b8:	38 a0 00 00 	li      r5,0
-  bc:	7c 09 03 a6 	mtctr   r0
-  c0:	80 db 01 54 	lwz     r6,340(r27)
-  c4:	3b 03 00 0c 	addi    r24,r3,12
-  c8:	80 1f 01 54 	lwz     r0,340(r31)
-  cc:	3a e4 00 10 	addi    r23,r4,16
-  d0:	7d 86 1a 14 	add     r12,r6,r3
-  d4:	39 25 00 01 	addi    r9,r5,1
-  d8:	80 cc 00 00 	lwz     r6,0(r12)
-  dc:	7f 20 1a 14 	add     r25,r0,r3
-  e0:	80 0c 00 04 	lwz     r0,4(r12)
-  e4:	39 63 00 18 	addi    r11,r3,24
-  e8:	39 44 00 20 	addi    r10,r4,32
-  ec:	38 e5 00 02 	addi    r7,r5,2
-  f0:	90 d9 00 00 	stw     r6,0(r25)
-  f4:	39 03 00 24 	addi    r8,r3,36
-  f8:	90 19 00 04 	stw     r0,4(r25)
-  fc:	80 0c 00 08 	lwz     r0,8(r12)
- 100:	90 19 00 08 	stw     r0,8(r25)
- 104:	80 db 01 58 	lwz     r6,344(r27)
- 108:	80 1f 01 58 	lwz     r0,344(r31)
- 10c:	7d 86 22 14 	add     r12,r6,r4
- 110:	80 cc 00 00 	lwz     r6,0(r12)
- 114:	7f 20 22 14 	add     r25,r0,r4
- 118:	80 0c 00 04 	lwz     r0,4(r12)
- 11c:	90 d9 00 00 	stw     r6,0(r25)
- 120:	90 19 00 04 	stw     r0,4(r25)
- 124:	80 cc 00 08 	lwz     r6,8(r12)
- 128:	80 0c 00 0c 	lwz     r0,12(r12)
- 12c:	90 d9 00 08 	stw     r6,8(r25)
- 130:	90 19 00 0c 	stw     r0,12(r25)
- 134:	81 9b 02 a8 	lwz     r12,680(r27)
- 138:	80 df 02 a8 	lwz     r6,680(r31)
- 13c:	7c 0c 28 ae 	lbzx    r0,r12,r5
- 140:	7c 06 29 ae 	stbx    r0,r6,r5
- 144:	80 db 01 54 	lwz     r6,340(r27)
- 148:	80 1f 01 54 	lwz     r0,340(r31)
- 14c:	7d 86 c2 14 	add     r12,r6,r24
- 150:	80 cc 00 00 	lwz     r6,0(r12)
- 154:	7f 00 c2 14 	add     r24,r0,r24
- 158:	80 0c 00 04 	lwz     r0,4(r12)
- 15c:	90 d8 00 00 	stw     r6,0(r24)
- 160:	90 18 00 04 	stw     r0,4(r24)
- 164:	80 0c 00 08 	lwz     r0,8(r12)
- 168:	90 18 00 08 	stw     r0,8(r24)
- 16c:	80 db 01 58 	lwz     r6,344(r27)
- 170:	80 1f 01 58 	lwz     r0,344(r31)
- 174:	7d 86 ba 14 	add     r12,r6,r23
- 178:	80 cc 00 00 	lwz     r6,0(r12)
- 17c:	7e e0 ba 14 	add     r23,r0,r23
- 180:	80 0c 00 04 	lwz     r0,4(r12)
- 184:	90 d7 00 00 	stw     r6,0(r23)
- 188:	90 17 00 04 	stw     r0,4(r23)
- 18c:	80 cc 00 08 	lwz     r6,8(r12)
- 190:	80 0c 00 0c 	lwz     r0,12(r12)
- 194:	90 d7 00 08 	stw     r6,8(r23)
- 198:	90 17 00 0c 	stw     r0,12(r23)
- 19c:	81 9b 02 a8 	lwz     r12,680(r27)
- 1a0:	80 df 02 a8 	lwz     r6,680(r31)
- 1a4:	7c 0c 48 ae 	lbzx    r0,r12,r9
- 1a8:	7c 06 49 ae 	stbx    r0,r6,r9
- 1ac:	80 db 01 54 	lwz     r6,340(r27)
- 1b0:	80 1f 01 54 	lwz     r0,340(r31)
- 1b4:	7d 26 5a 14 	add     r9,r6,r11
- 1b8:	80 c9 00 00 	lwz     r6,0(r9)
- 1bc:	7d 60 5a 14 	add     r11,r0,r11
- 1c0:	80 09 00 04 	lwz     r0,4(r9)
- 1c4:	90 cb 00 00 	stw     r6,0(r11)
- 1c8:	90 0b 00 04 	stw     r0,4(r11)
- 1cc:	80 09 00 08 	lwz     r0,8(r9)
- 1d0:	90 0b 00 08 	stw     r0,8(r11)
- 1d4:	80 db 01 58 	lwz     r6,344(r27)
- 1d8:	80 1f 01 58 	lwz     r0,344(r31)
- 1dc:	7d 26 52 14 	add     r9,r6,r10
- 1e0:	80 c9 00 00 	lwz     r6,0(r9)
- 1e4:	7d 40 52 14 	add     r10,r0,r10
- 1e8:	80 09 00 04 	lwz     r0,4(r9)
- 1ec:	90 ca 00 00 	stw     r6,0(r10)
- 1f0:	90 0a 00 04 	stw     r0,4(r10)
- 1f4:	80 c9 00 08 	lwz     r6,8(r9)
- 1f8:	80 09 00 0c 	lwz     r0,12(r9)
- 1fc:	90 ca 00 08 	stw     r6,8(r10)
- 200:	90 0a 00 0c 	stw     r0,12(r10)
- 204:	81 3b 02 a8 	lwz     r9,680(r27)
- 208:	80 df 02 a8 	lwz     r6,680(r31)
- 20c:	7c 09 38 ae 	lbzx    r0,r9,r7
- 210:	7c 06 39 ae 	stbx    r0,r6,r7
- 214:	80 db 01 54 	lwz     r6,340(r27)
- 218:	80 1f 01 54 	lwz     r0,340(r31)
- 21c:	7c e6 42 14 	add     r7,r6,r8
- 220:	80 c7 00 00 	lwz     r6,0(r7)
- 224:	7d 00 42 14 	add     r8,r0,r8
- 228:	80 07 00 04 	lwz     r0,4(r7)
- 22c:	90 c8 00 00 	stw     r6,0(r8)
- 230:	90 08 00 04 	stw     r0,4(r8)
- 234:	80 07 00 08 	lwz     r0,8(r7)
- 238:	90 08 00 08 	stw     r0,8(r8)
- 23c:	80 db 01 58 	lwz     r6,344(r27)
- 240:	38 e4 00 30 	addi    r7,r4,48
- 244:	80 1f 01 58 	lwz     r0,344(r31)
- 248:	3b a5 00 03 	addi    r29,r5,3
- 24c:	7f 06 3a 14 	add     r24,r6,r7
- 250:	3b 23 00 30 	addi    r25,r3,48
- 254:	80 d8 00 00 	lwz     r6,0(r24)
- 258:	7e e0 3a 14 	add     r23,r0,r7
- 25c:	80 18 00 04 	lwz     r0,4(r24)
- 260:	3b 44 00 40 	addi    r26,r4,64
- 264:	39 45 00 04 	addi    r10,r5,4
- 268:	39 83 00 3c 	addi    r12,r3,60
- 26c:	90 d7 00 00 	stw     r6,0(r23)
- 270:	39 64 00 50 	addi    r11,r4,80
- 274:	38 e5 00 05 	addi    r7,r5,5
- 278:	39 23 00 48 	addi    r9,r3,72
- 27c:	90 17 00 04 	stw     r0,4(r23)
- 280:	39 04 00 60 	addi    r8,r4,96
- 284:	80 d8 00 08 	lwz     r6,8(r24)
- 288:	80 18 00 0c 	lwz     r0,12(r24)
- 28c:	90 d7 00 08 	stw     r6,8(r23)
- 290:	90 17 00 0c 	stw     r0,12(r23)
- 294:	83 1b 02 a8 	lwz     r24,680(r27)
- 298:	80 df 02 a8 	lwz     r6,680(r31)
- 29c:	7c 18 e8 ae 	lbzx    r0,r24,r29
- 2a0:	7c 06 e9 ae 	stbx    r0,r6,r29
- 2a4:	80 db 01 54 	lwz     r6,340(r27)
- 2a8:	80 1f 01 54 	lwz     r0,340(r31)
- 2ac:	7f a6 ca 14 	add     r29,r6,r25
- 2b0:	80 dd 00 00 	lwz     r6,0(r29)
- 2b4:	7f 20 ca 14 	add     r25,r0,r25
- 2b8:	80 1d 00 04 	lwz     r0,4(r29)
- 2bc:	90 d9 00 00 	stw     r6,0(r25)
- 2c0:	90 19 00 04 	stw     r0,4(r25)
- 2c4:	80 1d 00 08 	lwz     r0,8(r29)
- 2c8:	90 19 00 08 	stw     r0,8(r25)
- 2cc:	80 db 01 58 	lwz     r6,344(r27)
- 2d0:	80 1f 01 58 	lwz     r0,344(r31)
- 2d4:	7f a6 d2 14 	add     r29,r6,r26
- 2d8:	80 dd 00 00 	lwz     r6,0(r29)
- 2dc:	7f 40 d2 14 	add     r26,r0,r26
- 2e0:	80 1d 00 04 	lwz     r0,4(r29)
- 2e4:	90 da 00 00 	stw     r6,0(r26)
- 2e8:	90 1a 00 04 	stw     r0,4(r26)
- 2ec:	80 dd 00 08 	lwz     r6,8(r29)
- 2f0:	80 1d 00 0c 	lwz     r0,12(r29)
- 2f4:	90 da 00 08 	stw     r6,8(r26)
- 2f8:	90 1a 00 0c 	stw     r0,12(r26)
- 2fc:	83 bb 02 a8 	lwz     r29,680(r27)
- 300:	80 df 02 a8 	lwz     r6,680(r31)
- 304:	7c 1d 50 ae 	lbzx    r0,r29,r10
- 308:	7c 06 51 ae 	stbx    r0,r6,r10
- 30c:	80 db 01 54 	lwz     r6,340(r27)
- 310:	80 1f 01 54 	lwz     r0,340(r31)
- 314:	7d 46 62 14 	add     r10,r6,r12
- 318:	80 ca 00 00 	lwz     r6,0(r10)
- 31c:	7d 80 62 14 	add     r12,r0,r12
- 320:	80 0a 00 04 	lwz     r0,4(r10)
- 324:	90 cc 00 00 	stw     r6,0(r12)
- 328:	90 0c 00 04 	stw     r0,4(r12)
- 32c:	80 0a 00 08 	lwz     r0,8(r10)
- 330:	90 0c 00 08 	stw     r0,8(r12)
- 334:	80 db 01 58 	lwz     r6,344(r27)
- 338:	80 1f 01 58 	lwz     r0,344(r31)
- 33c:	7d 46 5a 14 	add     r10,r6,r11
- 340:	80 ca 00 00 	lwz     r6,0(r10)
- 344:	7d 60 5a 14 	add     r11,r0,r11
- 348:	80 0a 00 04 	lwz     r0,4(r10)
- 34c:	90 cb 00 00 	stw     r6,0(r11)
- 350:	90 0b 00 04 	stw     r0,4(r11)
- 354:	80 ca 00 08 	lwz     r6,8(r10)
- 358:	80 0a 00 0c 	lwz     r0,12(r10)
- 35c:	90 cb 00 08 	stw     r6,8(r11)
- 360:	90 0b 00 0c 	stw     r0,12(r11)
- 364:	81 5b 02 a8 	lwz     r10,680(r27)
- 368:	80 df 02 a8 	lwz     r6,680(r31)
- 36c:	7c 0a 38 ae 	lbzx    r0,r10,r7
- 370:	7c 06 39 ae 	stbx    r0,r6,r7
- 374:	80 db 01 54 	lwz     r6,340(r27)
- 378:	80 1f 01 54 	lwz     r0,340(r31)
- 37c:	7c e6 4a 14 	add     r7,r6,r9
- 380:	80 c7 00 00 	lwz     r6,0(r7)
- 384:	7d 20 4a 14 	add     r9,r0,r9
- 388:	80 07 00 04 	lwz     r0,4(r7)
- 38c:	90 c9 00 00 	stw     r6,0(r9)
- 390:	90 09 00 04 	stw     r0,4(r9)
- 394:	80 07 00 08 	lwz     r0,8(r7)
- 398:	90 09 00 08 	stw     r0,8(r9)
- 39c:	80 db 01 58 	lwz     r6,344(r27)
- 3a0:	80 1f 01 58 	lwz     r0,344(r31)
- 3a4:	7c e6 42 14 	add     r7,r6,r8
- 3a8:	80 c7 00 00 	lwz     r6,0(r7)
- 3ac:	7d 00 42 14 	add     r8,r0,r8
- 3b0:	80 07 00 04 	lwz     r0,4(r7)
- 3b4:	90 c8 00 00 	stw     r6,0(r8)
- 3b8:	90 08 00 04 	stw     r0,4(r8)
- 3bc:	80 c7 00 08 	lwz     r6,8(r7)
- 3c0:	80 07 00 0c 	lwz     r0,12(r7)
- 3c4:	90 c8 00 08 	stw     r6,8(r8)
- 3c8:	90 08 00 0c 	stw     r0,12(r8)
- 3cc:	80 db 02 a8 	lwz     r6,680(r27)
- 3d0:	39 05 00 06 	addi    r8,r5,6
- 3d4:	39 43 00 54 	addi    r10,r3,84
- 3d8:	39 24 00 70 	addi    r9,r4,112
- 3dc:	7c 06 40 ae 	lbzx    r0,r6,r8
- 3e0:	38 e5 00 07 	addi    r7,r5,7
- 3e4:	80 df 02 a8 	lwz     r6,680(r31)
- 3e8:	38 63 00 60 	addi    r3,r3,96
- 3ec:	38 84 00 80 	addi    r4,r4,128
- 3f0:	38 a5 00 08 	addi    r5,r5,8
- 3f4:	7c 06 41 ae 	stbx    r0,r6,r8
- 3f8:	80 db 01 54 	lwz     r6,340(r27)
- 3fc:	80 1f 01 54 	lwz     r0,340(r31)
- 400:	7d 06 52 14 	add     r8,r6,r10
- 404:	7d 40 52 14 	add     r10,r0,r10
- 408:	80 c8 00 00 	lwz     r6,0(r8)
- 40c:	80 08 00 04 	lwz     r0,4(r8)
- 410:	90 ca 00 00 	stw     r6,0(r10)
- 414:	90 0a 00 04 	stw     r0,4(r10)
- 418:	80 08 00 08 	lwz     r0,8(r8)
- 41c:	90 0a 00 08 	stw     r0,8(r10)
- 420:	80 db 01 58 	lwz     r6,344(r27)
- 424:	80 1f 01 58 	lwz     r0,344(r31)
- 428:	7d 06 4a 14 	add     r8,r6,r9
- 42c:	7d 20 4a 14 	add     r9,r0,r9
- 430:	80 c8 00 00 	lwz     r6,0(r8)
- 434:	80 08 00 04 	lwz     r0,4(r8)
- 438:	90 c9 00 00 	stw     r6,0(r9)
- 43c:	90 09 00 04 	stw     r0,4(r9)
- 440:	80 c8 00 08 	lwz     r6,8(r8)
- 444:	80 08 00 0c 	lwz     r0,12(r8)
- 448:	90 c9 00 08 	stw     r6,8(r9)
- 44c:	90 09 00 0c 	stw     r0,12(r9)
- 450:	81 1b 02 a8 	lwz     r8,680(r27)
- 454:	80 df 02 a8 	lwz     r6,680(r31)
- 458:	7c 08 38 ae 	lbzx    r0,r8,r7
- 45c:	7c 06 39 ae 	stbx    r0,r6,r7
- 460:	42 00 fc 60 	bdnz    c0 <fn_8012E568+0xc0>
- 464:	20 05 00 89 	subfic  r0,r5,137
- 468:	54 a8 20 36 	slwi    r8,r5,4
- 46c:	1c e5 00 0c 	mulli   r7,r5,12
- 470:	7c 09 03 a6 	mtctr   r0
- 474:	2c 05 00 89 	cmpwi   r5,137
- 478:	40 80 00 7c 	bge     4f4 <fn_8012E568+0x4f4>
- 47c:	80 7b 01 54 	lwz     r3,340(r27)
- 480:	80 1f 01 54 	lwz     r0,340(r31)
- 484:	7c 83 3a 14 	add     r4,r3,r7
- 488:	7c c0 3a 14 	add     r6,r0,r7
- 48c:	80 64 00 00 	lwz     r3,0(r4)
- 490:	80 04 00 04 	lwz     r0,4(r4)
- 494:	38 e7 00 0c 	addi    r7,r7,12
- 498:	90 66 00 00 	stw     r3,0(r6)
- 49c:	90 06 00 04 	stw     r0,4(r6)
- 4a0:	80 04 00 08 	lwz     r0,8(r4)
- 4a4:	90 06 00 08 	stw     r0,8(r6)
- 4a8:	80 7b 01 58 	lwz     r3,344(r27)
- 4ac:	80 1f 01 58 	lwz     r0,344(r31)
- 4b0:	7c 83 42 14 	add     r4,r3,r8
- 4b4:	7c c0 42 14 	add     r6,r0,r8
- 4b8:	80 64 00 00 	lwz     r3,0(r4)
- 4bc:	80 04 00 04 	lwz     r0,4(r4)
- 4c0:	39 08 00 10 	addi    r8,r8,16
- 4c4:	90 66 00 00 	stw     r3,0(r6)
- 4c8:	90 06 00 04 	stw     r0,4(r6)
- 4cc:	80 64 00 08 	lwz     r3,8(r4)
- 4d0:	80 04 00 0c 	lwz     r0,12(r4)
- 4d4:	90 66 00 08 	stw     r3,8(r6)
- 4d8:	90 06 00 0c 	stw     r0,12(r6)
- 4dc:	80 9b 02 a8 	lwz     r4,680(r27)
- 4e0:	80 7f 02 a8 	lwz     r3,680(r31)
- 4e4:	7c 04 28 ae 	lbzx    r0,r4,r5
- 4e8:	7c 03 29 ae 	stbx    r0,r3,r5
- 4ec:	38 a5 00 01 	addi    r5,r5,1
- 4f0:	42 00 ff 8c 	bdnz    47c <fn_8012E568+0x47c>
- 4f4:	a0 9e 00 0e 	lhz     r4,14(r30)
- 4f8:	7f 63 db 78 	mr      r3,r27
- 4fc:	38 a1 00 14 	addi    r5,r1,20
- 500:	48 00 00 01 	bl      500 <fn_8012E568+0x500>
- 504:	38 61 00 14 	addi    r3,r1,20
- 508:	7f 64 db 78 	mr      r4,r27
- 50c:	7c 65 1b 78 	mr      r5,r3
- 510:	48 00 00 01 	bl      510 <fn_8012E568+0x510>
- 514:	2c 1c 00 02 	cmpwi   r28,2
- 518:	c0 20 00 00 	lfs     f1,0(0)
- 51c:	41 82 00 34 	beq     550 <fn_8012E568+0x550>
- 520:	40 80 00 14 	bge     534 <fn_8012E568+0x534>
- 524:	2c 1c 00 00 	cmpwi   r28,0
- 528:	41 82 00 18 	beq     540 <fn_8012E568+0x540>
- 52c:	40 80 00 34 	bge     560 <fn_8012E568+0x560>
- 530:	48 00 00 44 	b       574 <fn_8012E568+0x574>
- 534:	2c 1c 00 04 	cmpwi   r28,4
- 538:	40 80 00 3c 	bge     574 <fn_8012E568+0x574>
- 53c:	48 00 00 1c 	b       558 <fn_8012E568+0x558>
- 540:	c0 01 00 1c 	lfs     f0,28(r1)
- 544:	ec 00 08 2a 	fadds   f0,f0,f1
- 548:	d0 01 00 1c 	stfs    f0,28(r1)
- 54c:	48 00 00 28 	b       574 <fn_8012E568+0x574>
- 550:	c0 20 00 00 	lfs     f1,0(0)
- 554:	48 00 00 20 	b       574 <fn_8012E568+0x574>
- 558:	c0 20 00 00 	lfs     f1,0(0)
- 55c:	48 00 00 18 	b       574 <fn_8012E568+0x574>
- 560:	c0 00 00 00 	lfs     f0,0(0)
- 564:	c0 41 00 1c 	lfs     f2,28(r1)
- 568:	fc 20 00 90 	fmr     f1,f0
- 56c:	ec 02 00 2a 	fadds   f0,f2,f0
- 570:	d0 01 00 1c 	stfs    f0,28(r1)
- 574:	7f e3 fb 78 	mr      r3,r31
- 578:	48 00 00 01 	bl      578 <fn_8012E568+0x578>
- 57c:	80 c1 00 14 	lwz     r6,20(r1)
- 580:	7f e3 fb 78 	mr      r3,r31
- 584:	80 a1 00 18 	lwz     r5,24(r1)
- 588:	38 81 00 08 	addi    r4,r1,8
- 58c:	80 01 00 1c 	lwz     r0,28(r1)
- 590:	90 c1 00 08 	stw     r6,8(r1)
- 594:	90 a1 00 0c 	stw     r5,12(r1)
- 598:	90 01 00 10 	stw     r0,16(r1)
- 59c:	48 00 00 01 	bl      59c <fn_8012E568+0x59c>
- 5a0:	80 1f 02 54 	lwz     r0,596(r31)
- 5a4:	7f e3 fb 78 	mr      r3,r31
- 5a8:	60 00 04 00 	ori     r0,r0,1024
- 5ac:	90 1f 02 54 	stw     r0,596(r31)
- 5b0:	ba e1 00 2c 	lmw     r23,44(r1)
- 5b4:	80 01 00 54 	lwz     r0,84(r1)
- 5b8:	7c 08 03 a6 	mtlr    r0
- 5bc:	38 21 00 50 	addi    r1,r1,80
- 5c0:	4e 80 00 20 	blr
+   c:	39 61 00 50 	addi    r11,r1,80
+  10:	48 00 00 01 	bl      10 <fn_8012E568+0x10>
+  14:	7c 7f 1b 78 	mr      r31,r3
+  18:	80 63 00 3c 	lwz     r3,60(r3)
+  1c:	c0 3f 00 00 	lfs     f1,0(r31)
+  20:	7c 9b 23 78 	mr      r27,r4
+  24:	c0 5f 00 04 	lfs     f2,4(r31)
+  28:	7c bc 2b 78 	mr      r28,r5
+  2c:	c0 7f 00 08 	lfs     f3,8(r31)
+  30:	48 00 00 01 	bl      30 <fn_8012E568+0x30>
+  34:	80 9f 02 40 	lwz     r4,576(r31)
+  38:	57 60 10 3a 	slwi    r0,r27,2
+  3c:	7c 7e 1b 78 	mr      r30,r3
+  40:	7c 84 00 2e 	lwzx    r4,r4,r0
+  44:	83 a4 00 04 	lwz     r29,4(r4)
+  48:	48 00 00 01 	bl      48 <fn_8012E568+0x48>
+  4c:	80 1f 02 94 	lwz     r0,660(r31)
+  50:	7f e3 fb 78 	mr      r3,r31
+  54:	38 80 00 00 	li      r4,0
+  58:	90 1e 02 94 	stw     r0,660(r30)
+  5c:	48 00 00 01 	bl      5c <fn_8012E568+0x5c>
+  60:	80 1f 00 2c 	lwz     r0,44(r31)
+  64:	28 1c 00 00 	cmplwi  r28,0
+  68:	90 1e 00 2c 	stw     r0,44(r30)
+  6c:	80 1f 00 30 	lwz     r0,48(r31)
+  70:	90 1e 00 30 	stw     r0,48(r30)
+  74:	80 1f 00 34 	lwz     r0,52(r31)
+  78:	90 1e 00 34 	stw     r0,52(r30)
+  7c:	80 1f 00 38 	lwz     r0,56(r31)
+  80:	90 1e 00 38 	stw     r0,56(r30)
+  84:	80 1f 02 50 	lwz     r0,592(r31)
+  88:	90 1e 02 50 	stw     r0,592(r30)
+  8c:	80 1f 02 44 	lwz     r0,580(r31)
+  90:	90 1e 02 44 	stw     r0,580(r30)
+  94:	41 82 00 1c 	beq     b0 <fn_8012E568+0xb0>
+  98:	80 7c 00 00 	lwz     r3,0(r28)
+  9c:	80 1c 00 04 	lwz     r0,4(r28)
+  a0:	90 7e 00 0c 	stw     r3,12(r30)
+  a4:	90 1e 00 10 	stw     r0,16(r30)
+  a8:	80 1c 00 08 	lwz     r0,8(r28)
+  ac:	90 1e 00 14 	stw     r0,20(r30)
+  b0:	38 60 00 00 	li      r3,0
+  b4:	38 00 00 11 	li      r0,17
+  b8:	7c 64 1b 78 	mr      r4,r3
+  bc:	38 a0 00 00 	li      r5,0
+  c0:	7c 09 03 a6 	mtctr   r0
+  c4:	80 df 01 54 	lwz     r6,340(r31)
+  c8:	3b 03 00 0c 	addi    r24,r3,12
+  cc:	80 1e 01 54 	lwz     r0,340(r30)
+  d0:	3a e4 00 10 	addi    r23,r4,16
+  d4:	7d 86 1a 14 	add     r12,r6,r3
+  d8:	39 25 00 01 	addi    r9,r5,1
+  dc:	80 cc 00 00 	lwz     r6,0(r12)
+  e0:	7f 20 1a 14 	add     r25,r0,r3
+  e4:	80 0c 00 04 	lwz     r0,4(r12)
+  e8:	39 63 00 18 	addi    r11,r3,24
+  ec:	39 44 00 20 	addi    r10,r4,32
+  f0:	38 e5 00 02 	addi    r7,r5,2
+  f4:	90 d9 00 00 	stw     r6,0(r25)
+  f8:	39 03 00 24 	addi    r8,r3,36
+  fc:	90 19 00 04 	stw     r0,4(r25)
+ 100:	80 0c 00 08 	lwz     r0,8(r12)
+ 104:	90 19 00 08 	stw     r0,8(r25)
+ 108:	80 df 01 58 	lwz     r6,344(r31)
+ 10c:	80 1e 01 58 	lwz     r0,344(r30)
+ 110:	7d 86 22 14 	add     r12,r6,r4
+ 114:	80 cc 00 00 	lwz     r6,0(r12)
+ 118:	7f 20 22 14 	add     r25,r0,r4
+ 11c:	80 0c 00 04 	lwz     r0,4(r12)
+ 120:	90 d9 00 00 	stw     r6,0(r25)
+ 124:	90 19 00 04 	stw     r0,4(r25)
+ 128:	80 cc 00 08 	lwz     r6,8(r12)
+ 12c:	80 0c 00 0c 	lwz     r0,12(r12)
+ 130:	90 d9 00 08 	stw     r6,8(r25)
+ 134:	90 19 00 0c 	stw     r0,12(r25)
+ 138:	81 9f 02 a8 	lwz     r12,680(r31)
+ 13c:	80 de 02 a8 	lwz     r6,680(r30)
+ 140:	7c 0c 28 ae 	lbzx    r0,r12,r5
+ 144:	7c 06 29 ae 	stbx    r0,r6,r5
+ 148:	80 df 01 54 	lwz     r6,340(r31)
+ 14c:	80 1e 01 54 	lwz     r0,340(r30)
+ 150:	7d 86 c2 14 	add     r12,r6,r24
+ 154:	80 cc 00 00 	lwz     r6,0(r12)
+ 158:	7f 00 c2 14 	add     r24,r0,r24
+ 15c:	80 0c 00 04 	lwz     r0,4(r12)
+ 160:	90 d8 00 00 	stw     r6,0(r24)
+ 164:	90 18 00 04 	stw     r0,4(r24)
+ 168:	80 0c 00 08 	lwz     r0,8(r12)
+ 16c:	90 18 00 08 	stw     r0,8(r24)
+ 170:	80 df 01 58 	lwz     r6,344(r31)
+ 174:	80 1e 01 58 	lwz     r0,344(r30)
+ 178:	7d 86 ba 14 	add     r12,r6,r23
+ 17c:	80 cc 00 00 	lwz     r6,0(r12)
+ 180:	7e e0 ba 14 	add     r23,r0,r23
+ 184:	80 0c 00 04 	lwz     r0,4(r12)
+ 188:	90 d7 00 00 	stw     r6,0(r23)
+ 18c:	90 17 00 04 	stw     r0,4(r23)
+ 190:	80 cc 00 08 	lwz     r6,8(r12)
+ 194:	80 0c 00 0c 	lwz     r0,12(r12)
+ 198:	90 d7 00 08 	stw     r6,8(r23)
+ 19c:	90 17 00 0c 	stw     r0,12(r23)
+ 1a0:	81 9f 02 a8 	lwz     r12,680(r31)
+ 1a4:	80 de 02 a8 	lwz     r6,680(r30)
+ 1a8:	7c 0c 48 ae 	lbzx    r0,r12,r9
+ 1ac:	7c 06 49 ae 	stbx    r0,r6,r9
+ 1b0:	80 df 01 54 	lwz     r6,340(r31)
+ 1b4:	80 1e 01 54 	lwz     r0,340(r30)
+ 1b8:	7d 26 5a 14 	add     r9,r6,r11
+ 1bc:	80 c9 00 00 	lwz     r6,0(r9)
+ 1c0:	7d 60 5a 14 	add     r11,r0,r11
+ 1c4:	80 09 00 04 	lwz     r0,4(r9)
+ 1c8:	90 cb 00 00 	stw     r6,0(r11)
+ 1cc:	90 0b 00 04 	stw     r0,4(r11)
+ 1d0:	80 09 00 08 	lwz     r0,8(r9)
+ 1d4:	90 0b 00 08 	stw     r0,8(r11)
+ 1d8:	80 df 01 58 	lwz     r6,344(r31)
+ 1dc:	80 1e 01 58 	lwz     r0,344(r30)
+ 1e0:	7d 26 52 14 	add     r9,r6,r10
+ 1e4:	80 c9 00 00 	lwz     r6,0(r9)
+ 1e8:	7d 40 52 14 	add     r10,r0,r10
+ 1ec:	80 09 00 04 	lwz     r0,4(r9)
+ 1f0:	90 ca 00 00 	stw     r6,0(r10)
+ 1f4:	90 0a 00 04 	stw     r0,4(r10)
+ 1f8:	80 c9 00 08 	lwz     r6,8(r9)
+ 1fc:	80 09 00 0c 	lwz     r0,12(r9)
+ 200:	90 ca 00 08 	stw     r6,8(r10)
+ 204:	90 0a 00 0c 	stw     r0,12(r10)
+ 208:	81 3f 02 a8 	lwz     r9,680(r31)
+ 20c:	80 de 02 a8 	lwz     r6,680(r30)
+ 210:	7c 09 38 ae 	lbzx    r0,r9,r7
+ 214:	7c 06 39 ae 	stbx    r0,r6,r7
+ 218:	80 df 01 54 	lwz     r6,340(r31)
+ 21c:	80 1e 01 54 	lwz     r0,340(r30)
+ 220:	7c e6 42 14 	add     r7,r6,r8
+ 224:	80 c7 00 00 	lwz     r6,0(r7)
+ 228:	7d 00 42 14 	add     r8,r0,r8
+ 22c:	80 07 00 04 	lwz     r0,4(r7)
+ 230:	90 c8 00 00 	stw     r6,0(r8)
+ 234:	90 08 00 04 	stw     r0,4(r8)
+ 238:	80 07 00 08 	lwz     r0,8(r7)
+ 23c:	90 08 00 08 	stw     r0,8(r8)
+ 240:	80 df 01 58 	lwz     r6,344(r31)
+ 244:	38 e4 00 30 	addi    r7,r4,48
+ 248:	80 1e 01 58 	lwz     r0,344(r30)
+ 24c:	3b 85 00 03 	addi    r28,r5,3
+ 250:	7f 06 3a 14 	add     r24,r6,r7
+ 254:	3b 23 00 30 	addi    r25,r3,48
+ 258:	80 d8 00 00 	lwz     r6,0(r24)
+ 25c:	7e e0 3a 14 	add     r23,r0,r7
+ 260:	80 18 00 04 	lwz     r0,4(r24)
+ 264:	3b 44 00 40 	addi    r26,r4,64
+ 268:	39 45 00 04 	addi    r10,r5,4
+ 26c:	39 83 00 3c 	addi    r12,r3,60
+ 270:	90 d7 00 00 	stw     r6,0(r23)
+ 274:	39 64 00 50 	addi    r11,r4,80
+ 278:	38 e5 00 05 	addi    r7,r5,5
+ 27c:	39 23 00 48 	addi    r9,r3,72
+ 280:	90 17 00 04 	stw     r0,4(r23)
+ 284:	39 04 00 60 	addi    r8,r4,96
+ 288:	80 d8 00 08 	lwz     r6,8(r24)
+ 28c:	80 18 00 0c 	lwz     r0,12(r24)
+ 290:	90 d7 00 08 	stw     r6,8(r23)
+ 294:	90 17 00 0c 	stw     r0,12(r23)
+ 298:	83 1f 02 a8 	lwz     r24,680(r31)
+ 29c:	80 de 02 a8 	lwz     r6,680(r30)
+ 2a0:	7c 18 e0 ae 	lbzx    r0,r24,r28
+ 2a4:	7c 06 e1 ae 	stbx    r0,r6,r28
+ 2a8:	80 df 01 54 	lwz     r6,340(r31)
+ 2ac:	80 1e 01 54 	lwz     r0,340(r30)
+ 2b0:	7f 86 ca 14 	add     r28,r6,r25
+ 2b4:	80 dc 00 00 	lwz     r6,0(r28)
+ 2b8:	7f 20 ca 14 	add     r25,r0,r25
+ 2bc:	80 1c 00 04 	lwz     r0,4(r28)
+ 2c0:	90 d9 00 00 	stw     r6,0(r25)
+ 2c4:	90 19 00 04 	stw     r0,4(r25)
+ 2c8:	80 1c 00 08 	lwz     r0,8(r28)
+ 2cc:	90 19 00 08 	stw     r0,8(r25)
+ 2d0:	80 df 01 58 	lwz     r6,344(r31)
+ 2d4:	80 1e 01 58 	lwz     r0,344(r30)
+ 2d8:	7f 86 d2 14 	add     r28,r6,r26
+ 2dc:	80 dc 00 00 	lwz     r6,0(r28)
+ 2e0:	7f 40 d2 14 	add     r26,r0,r26
+ 2e4:	80 1c 00 04 	lwz     r0,4(r28)
+ 2e8:	90 da 00 00 	stw     r6,0(r26)
+ 2ec:	90 1a 00 04 	stw     r0,4(r26)
+ 2f0:	80 dc 00 08 	lwz     r6,8(r28)
+ 2f4:	80 1c 00 0c 	lwz     r0,12(r28)
+ 2f8:	90 da 00 08 	stw     r6,8(r26)
+ 2fc:	90 1a 00 0c 	stw     r0,12(r26)
+ 300:	83 9f 02 a8 	lwz     r28,680(r31)
+ 304:	80 de 02 a8 	lwz     r6,680(r30)
+ 308:	7c 1c 50 ae 	lbzx    r0,r28,r10
+ 30c:	7c 06 51 ae 	stbx    r0,r6,r10
+ 310:	80 df 01 54 	lwz     r6,340(r31)
+ 314:	80 1e 01 54 	lwz     r0,340(r30)
+ 318:	7d 46 62 14 	add     r10,r6,r12
+ 31c:	80 ca 00 00 	lwz     r6,0(r10)
+ 320:	7d 80 62 14 	add     r12,r0,r12
+ 324:	80 0a 00 04 	lwz     r0,4(r10)
+ 328:	90 cc 00 00 	stw     r6,0(r12)
+ 32c:	90 0c 00 04 	stw     r0,4(r12)
+ 330:	80 0a 00 08 	lwz     r0,8(r10)
+ 334:	90 0c 00 08 	stw     r0,8(r12)
+ 338:	80 df 01 58 	lwz     r6,344(r31)
+ 33c:	80 1e 01 58 	lwz     r0,344(r30)
+ 340:	7d 46 5a 14 	add     r10,r6,r11
+ 344:	80 ca 00 00 	lwz     r6,0(r10)
+ 348:	7d 60 5a 14 	add     r11,r0,r11
+ 34c:	80 0a 00 04 	lwz     r0,4(r10)
+ 350:	90 cb 00 00 	stw     r6,0(r11)
+ 354:	90 0b 00 04 	stw     r0,4(r11)
+ 358:	80 ca 00 08 	lwz     r6,8(r10)
+ 35c:	80 0a 00 0c 	lwz     r0,12(r10)
+ 360:	90 cb 00 08 	stw     r6,8(r11)
+ 364:	90 0b 00 0c 	stw     r0,12(r11)
+ 368:	81 5f 02 a8 	lwz     r10,680(r31)
+ 36c:	80 de 02 a8 	lwz     r6,680(r30)
+ 370:	7c 0a 38 ae 	lbzx    r0,r10,r7
+ 374:	7c 06 39 ae 	stbx    r0,r6,r7
+ 378:	80 df 01 54 	lwz     r6,340(r31)
+ 37c:	80 1e 01 54 	lwz     r0,340(r30)
+ 380:	7c e6 4a 14 	add     r7,r6,r9
+ 384:	80 c7 00 00 	lwz     r6,0(r7)
+ 388:	7d 20 4a 14 	add     r9,r0,r9
+ 38c:	80 07 00 04 	lwz     r0,4(r7)
+ 390:	90 c9 00 00 	stw     r6,0(r9)
+ 394:	90 09 00 04 	stw     r0,4(r9)
+ 398:	80 07 00 08 	lwz     r0,8(r7)
+ 39c:	90 09 00 08 	stw     r0,8(r9)
+ 3a0:	80 df 01 58 	lwz     r6,344(r31)
+ 3a4:	80 1e 01 58 	lwz     r0,344(r30)
+ 3a8:	7c e6 42 14 	add     r7,r6,r8
+ 3ac:	80 c7 00 00 	lwz     r6,0(r7)
+ 3b0:	7d 00 42 14 	add     r8,r0,r8
+ 3b4:	80 07 00 04 	lwz     r0,4(r7)
+ 3b8:	90 c8 00 00 	stw     r6,0(r8)
+ 3bc:	90 08 00 04 	stw     r0,4(r8)
+ 3c0:	80 c7 00 08 	lwz     r6,8(r7)
+ 3c4:	80 07 00 0c 	lwz     r0,12(r7)
+ 3c8:	90 c8 00 08 	stw     r6,8(r8)
+ 3cc:	90 08 00 0c 	stw     r0,12(r8)
+ 3d0:	80 df 02 a8 	lwz     r6,680(r31)
+ 3d4:	39 05 00 06 	addi    r8,r5,6
+ 3d8:	39 43 00 54 	addi    r10,r3,84
+ 3dc:	39 24 00 70 	addi    r9,r4,112
+ 3e0:	7c 06 40 ae 	lbzx    r0,r6,r8
+ 3e4:	38 e5 00 07 	addi    r7,r5,7
+ 3e8:	80 de 02 a8 	lwz     r6,680(r30)
+ 3ec:	38 63 00 60 	addi    r3,r3,96
+ 3f0:	38 84 00 80 	addi    r4,r4,128
+ 3f4:	38 a5 00 08 	addi    r5,r5,8
+ 3f8:	7c 06 41 ae 	stbx    r0,r6,r8
+ 3fc:	80 df 01 54 	lwz     r6,340(r31)
+ 400:	80 1e 01 54 	lwz     r0,340(r30)
+ 404:	7d 06 52 14 	add     r8,r6,r10
+ 408:	7d 40 52 14 	add     r10,r0,r10
+ 40c:	80 c8 00 00 	lwz     r6,0(r8)
+ 410:	80 08 00 04 	lwz     r0,4(r8)
+ 414:	90 ca 00 00 	stw     r6,0(r10)
+ 418:	90 0a 00 04 	stw     r0,4(r10)
+ 41c:	80 08 00 08 	lwz     r0,8(r8)
+ 420:	90 0a 00 08 	stw     r0,8(r10)
+ 424:	80 df 01 58 	lwz     r6,344(r31)
+ 428:	80 1e 01 58 	lwz     r0,344(r30)
+ 42c:	7d 06 4a 14 	add     r8,r6,r9
+ 430:	7d 20 4a 14 	add     r9,r0,r9
+ 434:	80 c8 00 00 	lwz     r6,0(r8)
+ 438:	80 08 00 04 	lwz     r0,4(r8)
+ 43c:	90 c9 00 00 	stw     r6,0(r9)
+ 440:	90 09 00 04 	stw     r0,4(r9)
+ 444:	80 c8 00 08 	lwz     r6,8(r8)
+ 448:	80 08 00 0c 	lwz     r0,12(r8)
+ 44c:	90 c9 00 08 	stw     r6,8(r9)
+ 450:	90 09 00 0c 	stw     r0,12(r9)
+ 454:	81 1f 02 a8 	lwz     r8,680(r31)
+ 458:	80 de 02 a8 	lwz     r6,680(r30)
+ 45c:	7c 08 38 ae 	lbzx    r0,r8,r7
+ 460:	7c 06 39 ae 	stbx    r0,r6,r7
+ 464:	42 00 fc 60 	bdnz    c4 <fn_8012E568+0xc4>
+ 468:	20 05 00 89 	subfic  r0,r5,137
+ 46c:	54 a8 20 36 	slwi    r8,r5,4
+ 470:	1c e5 00 0c 	mulli   r7,r5,12
+ 474:	7c 09 03 a6 	mtctr   r0
+ 478:	2c 05 00 89 	cmpwi   r5,137
+ 47c:	40 80 00 7c 	bge     4f8 <fn_8012E568+0x4f8>
+ 480:	80 7f 01 54 	lwz     r3,340(r31)
+ 484:	80 1e 01 54 	lwz     r0,340(r30)
+ 488:	7c 83 3a 14 	add     r4,r3,r7
+ 48c:	7c c0 3a 14 	add     r6,r0,r7
+ 490:	80 64 00 00 	lwz     r3,0(r4)
+ 494:	80 04 00 04 	lwz     r0,4(r4)
+ 498:	38 e7 00 0c 	addi    r7,r7,12
+ 49c:	90 66 00 00 	stw     r3,0(r6)
+ 4a0:	90 06 00 04 	stw     r0,4(r6)
+ 4a4:	80 04 00 08 	lwz     r0,8(r4)
+ 4a8:	90 06 00 08 	stw     r0,8(r6)
+ 4ac:	80 7f 01 58 	lwz     r3,344(r31)
+ 4b0:	80 1e 01 58 	lwz     r0,344(r30)
+ 4b4:	7c 83 42 14 	add     r4,r3,r8
+ 4b8:	7c c0 42 14 	add     r6,r0,r8
+ 4bc:	80 64 00 00 	lwz     r3,0(r4)
+ 4c0:	80 04 00 04 	lwz     r0,4(r4)
+ 4c4:	39 08 00 10 	addi    r8,r8,16
+ 4c8:	90 66 00 00 	stw     r3,0(r6)
+ 4cc:	90 06 00 04 	stw     r0,4(r6)
+ 4d0:	80 64 00 08 	lwz     r3,8(r4)
+ 4d4:	80 04 00 0c 	lwz     r0,12(r4)
+ 4d8:	90 66 00 08 	stw     r3,8(r6)
+ 4dc:	90 06 00 0c 	stw     r0,12(r6)
+ 4e0:	80 9f 02 a8 	lwz     r4,680(r31)
+ 4e4:	80 7e 02 a8 	lwz     r3,680(r30)
+ 4e8:	7c 04 28 ae 	lbzx    r0,r4,r5
+ 4ec:	7c 03 29 ae 	stbx    r0,r3,r5
+ 4f0:	38 a5 00 01 	addi    r5,r5,1
+ 4f4:	42 00 ff 8c 	bdnz    480 <fn_8012E568+0x480>
+ 4f8:	a0 9d 00 0e 	lhz     r4,14(r29)
+ 4fc:	7f e3 fb 78 	mr      r3,r31
+ 500:	38 a1 00 14 	addi    r5,r1,20
+ 504:	48 00 00 01 	bl      504 <fn_8012E568+0x504>
+ 508:	c0 41 00 14 	lfs     f2,20(r1)
+ 50c:	38 61 00 08 	addi    r3,r1,8
+ 510:	c0 21 00 18 	lfs     f1,24(r1)
+ 514:	7f e4 fb 78 	mr      r4,r31
+ 518:	c0 01 00 1c 	lfs     f0,28(r1)
+ 51c:	7c 65 1b 78 	mr      r5,r3
+ 520:	d0 41 00 08 	stfs    f2,8(r1)
+ 524:	d0 21 00 0c 	stfs    f1,12(r1)
+ 528:	d0 01 00 10 	stfs    f0,16(r1)
+ 52c:	48 00 00 01 	bl      52c <fn_8012E568+0x52c>
+ 530:	2c 1b 00 01 	cmpwi   r27,1
+ 534:	c0 20 00 00 	lfs     f1,0(0)
+ 538:	41 82 00 30 	beq     568 <fn_8012E568+0x568>
+ 53c:	40 80 00 10 	bge     54c <fn_8012E568+0x54c>
+ 540:	2c 1b 00 00 	cmpwi   r27,0
+ 544:	40 80 00 14 	bge     558 <fn_8012E568+0x558>
+ 548:	48 00 00 38 	b       580 <fn_8012E568+0x580>
+ 54c:	2c 1b 00 04 	cmpwi   r27,4
+ 550:	40 80 00 30 	bge     580 <fn_8012E568+0x580>
+ 554:	48 00 00 28 	b       57c <fn_8012E568+0x57c>
+ 558:	c0 01 00 10 	lfs     f0,16(r1)
+ 55c:	ec 00 08 2a 	fadds   f0,f0,f1
+ 560:	d0 01 00 10 	stfs    f0,16(r1)
+ 564:	48 00 00 1c 	b       580 <fn_8012E568+0x580>
+ 568:	c0 20 00 00 	lfs     f1,0(0)
+ 56c:	c0 01 00 10 	lfs     f0,16(r1)
+ 570:	ec 00 08 2a 	fadds   f0,f0,f1
+ 574:	d0 01 00 10 	stfs    f0,16(r1)
+ 578:	48 00 00 08 	b       580 <fn_8012E568+0x580>
+ 57c:	c0 20 00 00 	lfs     f1,0(0)
+ 580:	7f c3 f3 78 	mr      r3,r30
+ 584:	48 00 00 01 	bl      584 <fn_8012E568+0x584>
+ 588:	7f c3 f3 78 	mr      r3,r30
+ 58c:	38 81 00 08 	addi    r4,r1,8
+ 590:	48 00 00 01 	bl      590 <fn_8012E568+0x590>
+ 594:	80 1e 02 54 	lwz     r0,596(r30)
+ 598:	7f c3 f3 78 	mr      r3,r30
+ 59c:	60 00 04 00 	ori     r0,r0,1024
+ 5a0:	90 1e 02 54 	stw     r0,596(r30)
+ 5a4:	39 61 00 50 	addi    r11,r1,80
+ 5a8:	48 00 00 01 	bl      5a8 <fn_8012E568+0x5a8>
+ 5ac:	80 01 00 54 	lwz     r0,84(r1)
+ 5b0:	7c 08 03 a6 	mtlr    r0
+ 5b4:	38 21 00 50 	addi    r1,r1,80
+ 5b8:	4e 80 00 20 	blr

